WordPress/src/main/java/org/wordpress/android/widgets/PostListButton.java

```diff
@@ -139,9 +140,18 @@ public static int getButtonIconResId(int buttonType) {
             case BUTTON_BACK:
                 return R.drawable.ic_chevron_left_blue_wordpress_18dp;
             case BUTTON_RETRY:
-                return R.drawable.ic_refresh_blue_wordpress_18dp;
+                return R.drawable.ic_refresh_red_18dp;
             default:
                 return 0;
         }
     }
+
+    public static int getTextColorResId(int buttonType) {
```

Comment:
Minor tip: I've started using the `@StringRes`, `@DrawableRes` and `@ColorRes` annotations for these type of routines. ie: `public static @ColorRes int getTextColorResId(int buttonType)`

Reply:
Niiice! Added annotations in dd2caaf